ansys/mapdl/core/_commands/apdl/parameter_definition.py

```diff
@@ -295,36 +308,165 @@ def inquire(self, func):
 
         Notes
         -----
-        Allowable func entries
-        LOGIN - Returns the pathname of the login directory on Linux
-        systems or the pathname of the default directory (including
-        drive letter) on Windows systems.
-
-        - ``DOCU`` - Pathname of the ANSYS docu directory.
+        The ``/INQUIRE`` command is valid in any processor.
+
+        .. warning::
+           Take note that from version 0.60.4 and later, the previous command behaviour
+           has been changed.
+           Previously, the ``StrArray`` argument was omitted by mistake. For example:
+           >>> mapdl.inquire('DIRECTORY')
+           C:\\Users\\gayuso\\AppData\\Local\\Temp\\ansys_nynvxsaooh
+
+           Now this will raise an exception.
+           The default behaviour now, requires to input ``StrArray``:
+           >>> mapdl.inquire('', 'DIRECTORY')
+           C:\\Users\\gayuso\\AppData\\Local\\Temp\\ansys_nynvxsaooh
```

Comment:
```suggestion
           C:\\Users\\user\\AppData\\Local\\Temp\\ansys_nynvxsaooh

           Now this will raise an exception.
           The default behaviour now, requires to input ``StrArray``:
           >>> mapdl.inquire('', 'DIRECTORY')
           C:\\Users\\user\\AppData\\Local\\Temp\\ansys_nynvxsaooh
```
```suggestion
           C:\\Users\\gayuso\\AppData\\Local\\Temp\\ansys_nynvxsaooh

           Now this will raise an exception.
           The default behaviour now, requires to input ``StrArray``:
           >>> mapdl.inquire('', 'DIRECTORY')
           C:\\Users\\gayuso\\AppData\\Local\\Temp\\ansys_nynvxsaooh
```